.travis.yml

```diff
@@ -70,127 +60,112 @@ rvm:
 
 matrix:
   include:
-    - rvm: 2.5.3
-      env: "GEM=railties"
-      sudo: required
-      before_install:
-        - "rm ${BUNDLE_GEMFILE}.lock"
-        - "travis_retry gem update --system"
-        - "travis_retry gem install bundler"
-        - "sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/10/main/postgresql.conf"
-        - "sudo service postgresql restart 10"
-    - rvm: 2.6.0
-      env: "GEM=railties"
-      sudo: required
-      before_install:
-        - "rm ${BUNDLE_GEMFILE}.lock"
-        - "travis_retry gem update --system"
-        - "travis_retry gem install bundler"
-        - "sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/10/main/postgresql.conf"
-        - "sudo service postgresql restart 10"
-    - rvm: ruby-head
-      env: "GEM=railties"
-      sudo: required
-      before_install:
-        - "rm ${BUNDLE_GEMFILE}.lock"
-        - "travis_retry gem update --system"
-        - "travis_retry gem install bundler"
-        - "sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/10/main/postgresql.conf"
-        - "sudo service postgresql restart 10"
     - rvm: 2.5.3
       env: "GEM=actionview:ujs"
     - rvm: 2.5.3
       sudo: required
       env: "GEM=activejob:integration"
+      addons:
+        apt:
+          packages:
+          - rabbitmq-server
       services:
         - memcached
         - redis-server
-        - rabbitmq
+        - rabbitmq-server
       before_install:
-        - sudo sed -i -e '/local.*peer/s/postgres/all/' -e 's/peer\|md5/trust/g' /etc/postgresql/*/main/pg_hba.conf
-        - "sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/10/main/postgresql.conf"
         - "sudo service postgresql restart 10"
         - "[ -f /tmp/beanstalkd-1.10/Makefile ] || (curl -L https://github.com/beanstalkd/beanstalkd/archive/v1.10.tar.gz | tar xz -C /tmp)"
         - "pushd /tmp/beanstalkd-1.10 && make && (./beanstalkd &); popd"
     - rvm: 2.6.0
       sudo: required
       env: "GEM=activejob:integration"
+      addons:
+        apt:
+          packages:
+          - rabbitmq-server
       services:
         - memcached
         - redis-server
-        - rabbitmq
+        - rabbitmq-server
       before_install:
-        - sudo sed -i -e '/local.*peer/s/postgres/all/' -e 's/peer\|md5/trust/g' /etc/postgresql/*/main/pg_hba.conf
-        - "sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/10/main/postgresql.conf"
         - "sudo service postgresql restart 10"
```

Comment:
Is this restart necessary?

Reply:
I thik yes. But let me double check.

Reply:
Removing this line causes this error.  Added `postgresql: 10` to the each matrix also starts up PostgreSQL server, I think it is better than "sudo service postgresql restart 10" because 'restart" does not make sense.

https://travis-ci.org/rails/rails/jobs/481991504#L1220-L1226

```ruby
psql: could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
psql: could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
Cannot run integration tests for que. To be able to run integration tests for que you need to install and start postgresql.
```

Reply:
I see, thanks for the cheking.